src/backend/base/langflow/graph/graph/base.py

```diff
@@ -131,7 +131,8 @@ def activate_state_vertices(self, name: str, caller: str):
             name (str): The name of the state.
             caller (str): The ID of the vertex that is updating the state.
         """
-        vertices_ids = []
+        vertices_ids = set()
+        new_predecessor_map = {}
```

Comment:
If `new_predecessor_map` is from the `build_adjaceny_map()` method, would it be better to initialize the variable with a type be more specific?
```
new_predecessor_map: Dict[str, List[str]] = {}
```